riscv/src/compiler.rs

```diff
@@ -705,6 +735,10 @@ fn runtime() -> &'static str {
 
 .globl __rust_alloc_error_handler
 .set __rust_alloc_error_handler, __rg_oom
+
+.globl poseidon_coprocessor
+poseidon_coprocessor:
+    ret
```

Comment:
Why is this needed?

Reply:
because otherwise the linker will complain about a missing symbol:

```
The assembly code references an external routine / label that is not available:
runtime_dash_eee54d30a73402ee__poseidon_coprocessor
```